### ISSUE TITLE
Does not detect packages with a lower alpha numeric version

### ISSUE THREAD

**Issue by sarudak** [varchev/go-generic-artifactory-poller#15]
We often have multiple branches running in parallel development. The issue occurs for us when we have one open PR and then someone opens another PR for a different branch and then the original PR is updated creating a new artifact. The problem is the newest PR has a lower alpha numeric version number as generated by semver. Is there any way that we can set the plugin to grab every new artifact even if the version number is 'older' or setup the plugin to look at something else such as a timestamp on the artifact?

**Comment by sarudak** [comment]
After doing my own investigation it looks like this is not a realistic request given the GoCD and Artifactory APIs
